gradle/libs.versions.toml

```diff
@@ -55,7 +56,7 @@ spring-javaformat-checkstyle = { module = "io.spring.javaformat:spring-javaforma
 spring-javaformat-gradle-plugin = { module = "io.spring.javaformat:spring-javaformat-gradle-plugin", version.ref = "spring-javaformat" }
 licenser = { module = "gradle.plugin.org.cadixdev.gradle:licenser", version.ref = "licenser" }
 mockito-core = { module = "org.mockito:mockito-core", version.ref = "mockito" }
-mockito-inline = { module = "org.mockito:mockito-inline", version.ref = "mockito" }
+mockito-inline = { module = "org.mockito:mockito-inline", version.ref = "mockito-inline" }
```

Comment:
mockito-inline isn't needed with newer Mockito versions. It is included in mockito-core

Reply:
@lhotari I don't think it is
https://mvnrepository.com/artifact/org.mockito/mockito-core/5.8.0

https://github.com/mockito/mockito/issues/2877

Reply:
There's no need to depend on it any more. I also removed mockito-inline in apache/pulsar repository and all tests pass. Let's simply remove the dependency.

Reply:
uhu you were right! I added a commit

Reply:
Thanks for the contribution @nicoloboschi